docs/source/time_series.rst

```diff
@@ -98,6 +98,10 @@ Retrieve datapoints in time zone in pandas dataframe
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 .. automethod:: cognite.client._api.datapoints.DatapointsAPI.retrieve_dataframe_in_tz
 
+Iterate through datapoints in chunks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. automethod:: cognite.client._api.datapoints.DatapointsAPI.__call__
+
```

Comment:
Any ideas on how to make sphinx make this pretty... hmm

Reply:
Tried a quick experiment in; https://github.com/cognitedata/cognite-sdk-python/pull/2015

My interpretation as a developer you should know that `__call__` and `__iter__` are dunder methods and how to call them. The moment we are talking about iterating in chunks, we are talking about a more advanced case and we expects more from the user.

Reply:
Interesting! As a developer, I really liked just having the functions listed directly (like in your experiment)! I do wonder if normal users prefer the "old way" with more descriptions though... But I take it you're okay with `__call__` just being that in the docs?

Reply:
Yes, I am absolutely okay with `__call__`. I agree with you as well that I like having the functions listed directly. It has the advantage that I can lookup the name of the function directly instead of having to know what it is called in the API.